packages/manager/apps/cloud/client/app/dbaas/logs/detail/index/add/logs-index-add.controller.js

```diff
@@ -28,21 +30,75 @@ class LogsIndexAddModalCtrl {
       this.populateIndex();
     } else {
       this.clearIndex();
+      this.accountDetails.load().then(() => {
+        this.ovhSubsidiary = this.accountDetails.data.me.ovhSubsidiary;
+        this.$q.all([this.mainOffer.load(), this.catalog.load()]).then(() => {
+          const selectedCatalog = this.catalog.data.plans.find(
+            (plan) => plan.planCode === this.mainOffer.data.planCode,
+          );
+          const selectedFamily = selectedCatalog.addonsFamily.find(
+            (addon) => addon.family === this.LogsConstants.ADD_ON_FAMILY.NEW,
+          );
+          const indexShard = selectedFamily.addons.find(
+            (add) =>
+              add.plan.planCode ===
+              this.LogsConstants.CONSUMPTION_REFERENCE.NB_SHARD,
+          );
+          const indexVolume = selectedFamily.addons.find(
+            (add) =>
+              add.plan.planCode ===
+              this.LogsConstants.CONSUMPTION_REFERENCE.INDEXED_DOCUMENTS,
+          );
+          this.indexPrice.shard.price =
+            indexShard.plan.details.pricings.default[0].price.text;
+          this.indexPrice.volume.price =
+            indexVolume.plan.details.pricings.default[0].price.text;
+        });
+      });
     }
   }
 
+  initLoaders() {
+    this.mainOffer = this.CucControllerHelper.request.getArrayLoader({
+      loaderFunction: () =>
+        this.LogsIndexService.getMainOffer(this.serviceName),
+    });
+    this.catalog = this.CucControllerHelper.request.getArrayLoader({
+      loaderFunction: () =>
+        this.LogsIndexService.getOrderCatalog(this.ovhSubsidiary),
+    });
+    this.accountDetails = this.CucControllerHelper.request.getHashLoader({
+      loaderFunction: () =>
+        this.LogsIndexService.getAccountDetails(this.serviceName),
+    });
+  }
+
+  showIndexPrices() {
+    return this.$translate.instant(
+      'logs_index_modal_nb_shard_help',
+      {
+        t0: this.indexPrice.shard.price,
+        t1: this.indexPrice.volume.price,
+      },
+      undefined,
+      false,
+      'sceParameters',
```

Comment:
Please add a comment as to why you do this 🙂

Reply:
Fixed